docs/maui/views/MediaElement.md

```diff
@@ -76,11 +76,65 @@ public class MainActivity : MauiAppCompatActivity
 }
 ```
 
+Edit the `androidManifest.xml` manifest file and add the following.
+
+```csharp
+<application android:allowBackup="true" android:icon="@mipmap/appicon" android:enableOnBackInvokedCallback="true" android:supportsRtl="true">
+    <service android:name="CommunityToolkit.Maui.Media.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
+      <intent-filter>
+        <action android:name="android.intent.action.MEDIA_BUTTON" />
+      </intent-filter>
+      <intent-filter>
+        <action android:name="androidx.media3.session.MediaSessionService"/>
+      </intent-filter>
+    </service>
+</application>
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
+<uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
+```
+
+> [!NOTE]
+> This modification to the Android manifest enables metadata display when playing a video. It provides support for notifications and is essential
+ for notifications to function across all relevant APIs. The change introduces a service and grants necessary permissions.
+
 For a full example of this method included in an application please refer to the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs)
 
-### [iOS/Mac Catalyst](#tab/macios)
+### [Mac Catalyst](#tab/mac)
 
-No setup is required.
+Edit the `Info.plist` for `MacCatalyst` and add the following keys.
+```csharp
+    <key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoadsInMedia</key>
+		<true/>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+	</dict>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>bluetooth-central</string>
```

Comment:
This is likely my own ignorance but why do we need to set `bluetooth-central`?

Reply:
I tested with and without. For iOS and mac each was the specific minimum for functionality to work. I spent many hours testing each possible scenario where I systematically tested this. I think it is about the classes I am using that affect how bluetooth behaves. In other areas of toolkit the same class is used and requires bluetooth. I think it is the override for the silent toggle switch on the side of the device. They use the same library for different reasons.